mta/tests/migration/controls/test_business_services.py

```diff
@@ -0,0 +1,25 @@
+import pytest
+from pytest_check import check
+
+from conftest import generate_description, generate_name
+
+
+@pytest.mark.business
+def test_create_business_service(business_service, stakeholder, get_api, delete_api, update_api):
+    # get business_service_from_db
+    business_service_from_db = get_api.businessservices_id_get(business_service.id)
+
+    # Assertion checking the created business services's name
+    check.equal(business_service_from_db.name, business_service.name)
+
+    # Edit the BS name and description
+    business_service.name = generate_name(start="update name")
+    business_service.description = generate_description(start="update description")
+    update_api.businessservices_id_put(business_service.id, business_service.to_dict())
+    update_business_from_db = get_api.businessservices_id_get(business_service.id)
```

Comment:
Please rename to `update_business_service_from_db`

Reply:
Done